sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileSystemAsyncClient.java

```diff
@@ -158,6 +160,22 @@ public DataLakeDirectoryAsyncClient getDirectoryAsyncClient(String directoryName
             blockBlobAsyncClient);
     }
 
+    /**
+     * Creates a new DataLakeDirectoryAsyncClient object by concatenating "" to the end of
+     * DataLakeFileSystemAsyncClient's URL. The new DataLakeDirectoryAsyncClient uses the same request policy pipeline
+     * as the DataLakeFileSystemAsyncClient.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * {@codesnippet com.azure.storage.file.datalake.DataLakeFileSystemAsyncClient.getRootDirectoryAsyncClient}
```

Comment:
Does this code snippet exist?

Reply:
oh I think I forgot to add the code snippets